Week1/README.md

```diff
@@ -31,6 +31,9 @@ Only watch the below chapters:
 
 - Helpful resource: http://jsbooks.revolunet.com/ (here you can find tons of free JavaScript books online)
 
+-You are here: Home / Learn To Program / The 50 Best Websites to Learn JavaScript
```

Comment:
hi @Kemirdin thanks for your contribution. Can you delete this line? I don't get why it says "You are here: Home / Learn To Program / " besides that i think it is a good resource to add. I you modify it I will merge your PR